tests/manual_test_async_tx.py

```diff
@@ -0,0 +1,101 @@
+# This file is part of Maker Keeper Framework.
+#
+# Copyright (C) 2020 EdNoepel
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import asyncio
+import logging
+import sys
+import threading
+import time
+
+from web3 import Web3, HTTPProvider
+
+from pymaker import Address
+from pymaker.deployment import DssDeployment
+from pymaker.gas import FixedGasPrice
+from pymaker.keys import register_keys
+from pymaker.numeric import Wad
+
+logging.basicConfig(format='%(asctime)-15s %(levelname)-8s %(message)s', level=logging.DEBUG)
+# reduce logspew
+logging.getLogger('urllib3').setLevel(logging.INFO)
+logging.getLogger("web3").setLevel(logging.INFO)
+logging.getLogger("asyncio").setLevel(logging.INFO)
+logging.getLogger("requests").setLevel(logging.INFO)
+
+endpoint_uri = f"https://localhost:8545"
+web3 = Web3(HTTPProvider(endpoint_uri=endpoint_uri, request_kwargs={"timeout": 60}))
+web3.eth.defaultAccount = sys.argv[1]   # ex: 0x0000000000000000000000000000000aBcdef123
+register_keys(web3, [sys.argv[2]])      # ex: key_file=~keys/default-account.json,pass_file=~keys/default-account.pass
+
+mcd = DssDeployment.from_node(web3)
+our_address = Address(web3.eth.defaultAccount)
+
+collateral = mcd.collaterals['ETH-A']
+ilk = collateral.ilk
+collateral.approve(our_address)
+
+GWEI = 1000000000
+
+
+class TestApp:
```

Comment:
Much thanks for having a look at this.

1. This isn't a unit test; it's meant to manually test behavior on a real chain. 1000 Wei (not Gwei) won't be mined on Kovan or Mainnet.
2a( I pushed another commit with this assertion.
2b) `pymaker` doesn't seem to offer any facility allowing us to check whether a transaction was mined.  Should I implement such a facility?
3b) I'm doing cleanup because this manual test is meant to be run on a real chain.